pkg/tests/api/correlations/correlations_update_test.go

```diff
@@ -123,8 +128,12 @@ func TestIntegrationUpdateCorrelation(t *testing.T) {
 			url:  fmt.Sprintf("/api/datasources/uid/%s/correlations/%s", writableDs, "nonexistent-correlation-uid"),
 			user: adminUser,
 			body: `{
-				"label": ""
-			}`,
+				"label": "",
+				"config": {
+					"type": "query",
+					"field": "fieldName",
+					"target": { "expr": "bar" }
+				}}`,
```

Comment:
same, this change is not needed
```suggestion
				"label": ""
			}`,
```

Reply:
<img width="1172" alt="Screenshot 2022-10-07 at 11 03 00" src="https://user-images.githubusercontent.com/48948963/194516282-913165c8-a3d7-44c5-9010-2b35aa1df3dd.png">

For now it seems to be required but I will check again after I implemented your suggestion in pkg/services/correlations/models.go.

Reply:
i think you have an extra comma in your json
![Screenshot 2022-10-07 at 11 02 43](https://user-images.githubusercontent.com/1170767/194528674-67284f50-208a-430a-a897-2a867c1b8bb9.png)

Reply:
Yes. 🤦‍♀️ Removed it.